float32 voltage1
float32 voltage2
float32 voltage3